### DISCUSSION TITLE
While set Blazor Runtime to Webassembly in Oqtane 2.3, there are some render problem.

### DISCUSSION THREAD

**Discussion by Line4GitHub** [2sic/2sxc#2576] [General]
1. While set Blazor Runtime to Webassembly in Oqtane 2.3, there are some render problem.
2. The blank area on the right side of the Apps & Contents screen are too big.

**Comment by iJungleboy** [comment]
Could you share some more details, screenshot etc.?

**Comment by Line4GitHub** [comment]
https://github.com/oqtane/oqtane.framework/discussions/1748
1. While set Blazor Runtime to Webassembly in Oqtane 2.3 (Install with Oqtane.Framework.2.3.0.Install.zip),
2. 2sxc App & Content version:12.08.00 (Install with Module Management) there are some (ie. Blog 5.0 (Picture No Animation), Swiper Slider, FAQ App, Accordion) render problem.
3. Blog module for Oqtane version: 1.0.3, (Install with Module Management) after Installed & restart, The 2sxc App & Content are all encountered Error: System.AggregateException: One or more errors occurred. (Empty path name is not legal. (Parameter 'path')) ...
4. After Uninstall Blog module for Oqtane version: 1.0.3, there (2sxc App & Content) are no Error again.

**Reply by iJungleboy** [reply]
Ok this is a bit confusing. If I understand you correctly, the bug is because of the Blog 1.0.3 module, not because of 2sxc. Is that right?

**Reply by Line4GitHub** [reply]
Thank you very much for your reply!

The problem should be divided into (1&2), (3&4) two parts.
And
(3&4) not just Blog module for Oqtane version: 1.0.3, (Install with Module Management), also include StudioElf.Announcements  by:  Studio-Elf,,, etc. third party modules installed in Oqtane.

Thank you very much for your reply!

**Reply by iJungleboy** [reply]
ok, thx.
I'll convert this to an issue. 
@tvatavuk  could you look into this?

**Reply by Line4GitHub** [reply]
Thank you very much for your reply!

1. 2sxc apps & contents are very great and cool third-party module for Oqtane. and
2. I prefer the Blazor Wasm solutions.

Thank you very much for your reply!
